### ISSUE TITLE
ANN classifier - backpropagation

### ISSUE THREAD

**Issue by kggeorgieva** [kggeorgieva/ai_demo#3]
The aim of the task is to implement artificial neural network.
Please do not use a library with ready solution.
Demonstrate the implementation on one data sets selected by you.
You can use one data set from the following resources:
https://archive.ics.uci.edu/ml/index.php
https://www.kaggle.com/datasets
Calculate the accuracy of the algorithm using test set. You might use k-fold cross-validation.

Please find the trivia information about the method here:
https://en.wikipedia.org/wiki/Artificial_neural_network
https://en.wikipedia.org/wiki/Backpropagation
More thorough explanation here:
https://www.youtube.com/watch?v=uXt8qF2Zzfo
